src/components/visualizations/styles/tooltip.scss

```diff
@@ -180,3 +141,11 @@
         }
     }
 }
+
+// hide default highcharts svg tooltip when using html-based one
+.highcharts-tooltip-container {
+    g.highcharts-tooltip {
+        visibility: hidden;
+    }
+    z-index: 3005;
```

Comment:
Why 3005? This should be moved to _variables and add a comment to explain the number.

Reply:
i just keep the old one.
https://github.com/gooddata/gooddata-react-components/pull/1174/files#diff-84b5bf43cbbf7caee4c0bdabb659f0efL23